### ISSUE TITLE
4.7.3 fails to build with latest utf8-string

### ISSUE THREAD

**Issue by vmchale** [ddssff/listlike#10]
Build failure:

```
[13 of 21] Compiling Data.ListLike.UTF8 ( src/Data/ListLike/UTF8.hs, /tmp/ListLike-4.7.3/dist-newstyle/build/x86_64-linux/ghc-8.10.3/ListLike-4.7.3/build/Data/ListLike/UTF8.o, /tmp/ListLike-4.7.3/dist-newstyle/build/x86_64-linux/ghc-8.10.3/ListLike-4.7.3/build/Data/ListLike/UTF8.dyn_o )

src/Data/ListLike/UTF8.hs:153:10: error:
    • Overlapping instances for IsString (UTF8 BSC.ByteString)
        arising from the superclasses of an instance declaration
      Matching instances:
        instance UTF8.UTF8Bytes string index => IsString (UTF8 string)
          -- Defined in ‘Data.String.UTF8’
        instance IsString (UTF8 BSC.ByteString)
          -- Defined at src/Data/ListLike/UTF8.hs:150:10
    • In the instance declaration for
        ‘StringLike (UTF8 BSC.ByteString)’
    |
153 | instance StringLike (UTF8 BS.ByteString) where
    |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Data/ListLike/UTF8.hs:277:10: error:
    • Overlapping instances for IsString (UTF8 BSLC.ByteString)
        arising from the superclasses of an instance declaration
      Matching instances:
        instance UTF8.UTF8Bytes string index => IsString (UTF8 string)
          -- Defined in ‘Data.String.UTF8’
        instance IsString (UTF8 BSLC.ByteString)
          -- Defined at src/Data/ListLike/UTF8.hs:274:10
    • In the instance declaration for
        ‘StringLike (UTF8 BSLC.ByteString)’
    |
277 | instance StringLike (UTF8 BSL.ByteString) where
    |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

So constrained bounds or a new release would fix this! 

Thanks :)

**Comment by andreasabel** [comment]
I give it a try, fixing it in analogy to #7.  See PR #10.

**Comment by vmchale** [comment]
Thanks!

**Comment by ddssff** [comment]
Andreas, I've given you permissions in the listlike repository, enjoy!

**Comment by andreasabel** [comment]
Thanks @ddssff!  
I noticed that you released 4.7.3 on hackage.  If you give me up-load permission there, I can release 4.7.4.  I am AndreasAbel on hackage.

**Comment by andreasabel** [comment]
I suppose upload permission can be given here: https://hackage.haskell.org/package/ListLike/maintainers/
